frontend/src/pages/team/Devices/dialogs/TeamDeviceCreateDialog.vue

```diff
@@ -6,8 +6,14 @@
         <template #default>
             <slot name="description" />
             <form class="space-y-6 mt-2">
-                <FormRow v-model="input.name" data-form="device-name" :error="errors.name" :disabled="editDisabled">Name</FormRow>
-                <FormRow v-model="input.type" data-form="device-type" :error="errors.type" :disabled="editDisabled">Type</FormRow>
+                <FormRow v-model="input.name" data-form="device-name" :error="errors.name" :disabled="editDisabled" container-class="w-full">
+                    <template #default>Name</template>
+                    <template #description>Provide a unique, identifiable name for your device.</template>
+                </FormRow>
+                <FormRow v-model="input.type" data-form="device-type" :error="errors.type" :disabled="editDisabled" container-class="w-full">
+                    <template #default>Type</template>
+                    <template #description>Use this field to better identify your device, and sort/filter in your device list.</template>
```

Comment:
The type field becomes much more useful at the edge:

For example, you might deploy the same flow to 2 physically different device types.  The env var `FF_DEVICE_TYPE` can be used in switch nodes for routing messages, doing different things, accessing alternative edge connected things etc.

Reply:
didn't realise that `FF_DEVICE_TYPE` was an env var made available either. May be worth extending this description with something along those lines if you don't mind?